mysql-test/suite/innodb/t/stats_persistent.test

```diff
@@ -26,3 +27,13 @@ SET DEBUG_SYNC= 'RESET';
 DROP TABLE t1;
 
 --source include/wait_until_count_sessions.inc
+
+--echo #
+--echo # MDEV-36649 dict_acquire_mdl_shared() aborts when table
+--echo #      mode is DICT_TABLE_OP_OPEN_ONLY_IF_CACHED
+--echo #
+SET GLOBAL innodb_defragment_stats_accuracy=1;
+CREATE TABLE t (a INT ) ENGINE=INNODB;
+INSERT INTO t SELECT * FROM seq_1_to_1000;
+DROP TABLE t;
+SET GLOBAL innodb_defragment_stats_accuracy=default;
```

Comment:
I think that we should assign the original parameter to a `@save` variable and restore from it.